pybossa/importers/importer.py

```diff
@@ -177,12 +178,11 @@ def create_tasks(self, task_repo, project, **form_data):
             if found is None:
                 if validator.validate(task):
                     try:
-                        task_repo.save(task)
                         n += 1
+                        task_repo.save(task, clean_project=(n == n_tasks))
```

Comment:
if the csv import times out, or one of the tasks fails to save, the cache is not going to be cleared

Reply:
ok so I can call the clean from here then in the exception block.

Reply:
done